l10n_br_account/models/account_invoice.py

```diff
@@ -223,6 +223,8 @@ def copy(self, default=None):
         default = default or {}
         if self.document_type_id:
             default["line_ids"] = False
+        else:
+            default["line_ids"] = self.line_ids[0]
```

Comment:
```suggestion
        default["line_ids"] = False
```

Pelo que testei outra forma de resolver é retirar o if e o else e sempre setar o default["line_ids"] como False.

Reply:
Eu não testei desta forma. Vou testar ainda hoje.

Reply:
> Pelo que testei outra forma de resolver é retirar o if e o else e sempre setar o default["line_ids"] como False.

Esse aqui funcionou, 
ja o código da PR recebo o erro mencionado... @netosjb @marcelsavegnago